### PR TITLE
v4.0 pre-QA Copy Pass

### LINKED ISSUE #5239: VxAdmin: Copy incorrect in “Possible Double Vote Detected” warning
VxAdmin: If you try to adjudicate a write-in for a candidate that already has a vote, you see the “Possible Double Vote Detected” warning. It suggests selecting “Mark Write-in Invalid,” which isn’t one of the options.

(I think we changed that button to “Mark write-in as undervote,” but that also doesn’t apply here)

Issue automatically created from Slack feedback reported by jonah: 
https://votingworks.slack.com/archives/CJU9MSC6S/p1723072465437969?thread_ts=1723072465.437969&cid=CJU9MSC6S

### PR DESCRIPTION
**Pull request by adghayes**
## Overview

Closes #5239, #5203, and all current actions listed for #5141. I want to merge some of the known quick fixes and capitalization changes before QA because I have bandwidth and because we can be more focused when we review in QA.

## Demo Video or Screenshot

See commits. I don't have screenshots for everything, but enough to get. a sense of the changes.

### CODE REVIEW

File: apps/admin/frontend/src/screens/tally/import_cvrfiles_modal.tsx
Comment (adghayes) [inline apps/admin/frontend/src/screens/tally/import_cvrfiles_modal.tsx:1]: # Before
Screen Shot 2024-10-07 at 9 58 26 AM
Screen Shot 2024-10-07 at 9 58 59 AM

# After
Screen Shot 2024-10-07 at 9 59 13 AM
Screen Shot 2024-10-07 at 9 58 44 AM

File: apps/admin/frontend/src/components/export_election_package_modal_button.tsx [hunk 1]
```diff
@@ -109,9 +109,9 @@ export function ExportElectionPackageModalButton(): JSX.Element {
           title = 'Save Election Package';
           mainContent = (
             <P>
-              <UsbImage src="/assets/usb-drive.svg" alt="Insert USB Image" />A
-              zip archive will automatically be saved to the default location on
-              the mounted USB drive.
+              <UsbImage src="/assets/usb-drive.svg" alt="Insert USB Image" />
+              An election package will be saved to the default location on the
+              mounted USB drive.
```
Comment (adghayes) [inline apps/admin/frontend/src/components/export_election_package_modal_button.tsx:114]: # Before
Screen Shot 2024-10-07 at 10 41 06 AM
# After
Screen Shot 2024-10-07 at 10 45 04 AM

File: apps/mark-scan/frontend/src/pages/admin_screen.tsx [hunk 2]
```diff
@@ -137,10 +137,10 @@ export function AdminScreen({
                 </React.Fragment>
               )}
             </P>
-            <H6 as="h2">Test Ballot Mode</H6>
+            <H6 as="h2">Ballot Mode</H6>
             <P>
               <SegmentedButton
-                label="Test Ballot Mode"
+                label="Ballot Mode"
```
Comment (adghayes) [inline apps/mark-scan/frontend/src/pages/admin_screen.tsx:143]: # Before
Screen Shot 2024-10-07 at 11 06 11 AM

# After
Screen Shot 2024-10-07 at 11 06 55 AM

File: apps/mark-scan/frontend/src/pages/insert_card_screen.tsx [hunk 3]
```diff
@@ -43,7 +43,7 @@ export function InsertCardScreen({
         return (
           <React.Fragment>
             <H1>Polls Closed</H1>
-            <P>Insert Poll Worker card to open.</P>
```
Comment (adghayes) [inline apps/mark-scan/frontend/src/pages/insert_card_screen.tsx:46]: Screen Shot 2024-10-07 at 11 19 28 AM
Screen Shot 2024-10-07 at 11 20 12 AM

File: apps/mark-scan/frontend/src/pages/unconfigured_screen.tsx [hunk 4]
```diff
@@ -10,7 +10,7 @@ export function UnconfiguredScreen(): JSX.Element {
     <Screen>
       <Main centerChild padded>
         <FullScreenMessage
-          title="Insert an Election Manager card to configure VxMark"
```
Comment (adghayes) [inline apps/mark-scan/frontend/src/pages/unconfigured_screen.tsx:13]: Screen Shot 2024-10-07 at 11 28 19 AM
Screen Shot 2024-10-07 at 11 28 46 AM

File: apps/scan/frontend/src/app.test.tsx [hunk 5]
```diff
@@ -96,7 +96,7 @@ test('app can load and configure from a usb stick', async () => {
   apiMock.setPrinterStatusV4();
   renderApp();
   await screen.findByText(
-    'Insert an Election Manager card to configure VxScan'
+    'Insert an election manager card to configure VxScan'
```
Comment (adghayes) [inline apps/scan/frontend/src/app.test.tsx:99]: Screen Shot 2024-10-07 at 11 28 19 AM
Screen Shot 2024-10-07 at 11 28 46 AM

File: apps/admin/frontend/src/screens/machine_locked_screen.tsx [hunk 6]
```diff
@@ -22,8 +22,8 @@ export function MachineLockedScreen(): JSX.Element {
           <H1 align="center">VxAdmin is Locked</H1>
           <H3 style={{ fontWeight: 'normal' }}>
             {electionDefinition
-              ? 'Insert System Administrator or Election Manager card to unlock.'
-              : 'Insert System Administrator card to unlock.'}
+              ? 'Insert system administrator or election manager card to unlock.'
+              : 'Insert system administrator card to unlock.'}
```
Comment (adghayes) [inline apps/admin/frontend/src/screens/machine_locked_screen.tsx:26]: Screen Shot 2024-10-07 at 11 53 11 AM
Screen Shot 2024-10-07 at 11 55 29 AM

File: apps/admin/frontend/src/screens/smart_cards_screen.tsx [hunk 7]
```diff
@@ -131,7 +142,7 @@ function ActionResultCallout({ result }: { result: SmartCardActionResult }) {
     case 'Unprogram':
       return (
         <Callout color="primary" icon="Done">
-          {cardRole} card has been unprogrammed.
+          {toLowerCaseExceptFirst(cardRole)} card has been unprogrammed.
```
Comment (adghayes) [inline apps/admin/frontend/src/screens/smart_cards_screen.tsx:145]: Screen Shot 2024-10-07 at 11 51 28 AM
Screen Shot 2024-10-07 at 11 51 54 AM

File: apps/admin/frontend/src/screens/smart_cards_screen.test.tsx [hunk 8]
```diff
@@ -111,7 +111,7 @@ test('Insert card prompt', async () => {
   renderScreen({ apiMock });
 
   await screen.findByRole('heading', { name: 'Smart Cards' });
-  screen.getByText('Insert a Smart Card');
+  screen.getByText('Insert a smart card');
```
Comment (adghayes) [inline apps/admin/frontend/src/screens/smart_cards_screen.test.tsx:114]: Screen Shot 2024-10-07 at 12 25 15 PM
Screen Shot 2024-10-07 at 12 25 48 PM

File: apps/admin/frontend/src/screens/smart_cards_screen.tsx [hunk 9]
```diff
@@ -36,6 +36,18 @@ import {
   getAuthStatus,
 } from '../api';
 
+function toLowerCaseExceptFirst(str: string): string {
```
Comment (carolinemodic) [inline apps/admin/frontend/src/screens/smart_cards_screen.tsx:39]: could be nice to put this in /basics or /ui

**Review by carolinemodic — APPROVED** [review]

**Pull request merged**